net/interfaces/interfaces_bsd.go

```diff
@@ -37,85 +39,118 @@ func defaultRoute() (d DefaultRouteDetails, err error) {
 }
 
 func DefaultRouteInterfaceIndex() (int, error) {
```

Comment:
this is specifically for IPv4, right? docs while you're here?

I think elsewhere we have a TODO to add an address family arg to this if/when any caller needs to distinguish those. And IIRC we recently did need that but for Windows only so far.

Reply:
It does handle IPv6 though: https://github.com/tailscale/tailscale/blob/e7f9b92e1a394f4d5600feac223f79ed9c97d4ef/net/interfaces/interfaces_bsd.go#L175-L181

Added some docs.

Reply:
No, I mean: it's possible to configure machines such that their default route for IPv4 is a different interface from IPv6. This API doesn't support passing which the caller cares about. That's fine. No need to fix today. Just document whatever the behavior is